web/src/app/modules/views/main/editors/modules/editor-grid-button/components/editor-grid-button.component.ts

```diff
@@ -19,11 +19,13 @@ export class EditorGridButtonComponent {
   }
 
   public zoomIn(): void {
-    this.zoomFactor = this.zoomFactor * 1.1;
+    var zoomLimit: number = this.zoomFactor * 1.1;
+    this.zoomFactor = (zoomLimit <= 2) ? zoomLimit : this.zoomFactor;
   }
 
   public zoomOut(): void {
-    this.zoomFactor = this.zoomFactor / 1.1;
+    var zoomLimit: number = this.zoomFactor / 1.1;
```

Comment:
Looks like you are adjusting the zoom by 10% steps
I would be good to move the constant to the config file.

Reply:
Hi Eriks,

Thanks for your comment!

I limited the zoom level to work between 50% to 200%. The 10% step was already defined in the project and for the readability of the code, I have used the variable 'zoomlimit' to keep the value of this step.

Again thanks for your consideration and wishing you a great evening!

Best Regards,
Maryam Rezaei,